Update JsonDicomConverter.cs

nutzlastfan opened the pull request:
Added Encoding.Default for DicomIten creation VR types  LO, SH, PN
so special characters like umlaute would be deserialized properly.

Fixes # .

#### Checklist
- [x] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch
- [ ] I have updated API documentation
- [ ] I have included unit tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file

#### Changes proposed in this pull request:
-
-
-

gofal commented:
Thanks, missed those three VRs LO, PN and SH to add the encoding to the constrctor. But you should not take Encoding.Default - this results in different encodings depending on the local culture. 
DICOM standard F.2 defines that json text encoding shall always be UTF8. This class therefore already has the local variable '_jsonTextEncoding'. 

Would you please replace 'Encoding.Default' by '_jsonTextEncoding'? Then i will merge.

Thanks again for the pull request.

nutzlastfan commented:
Ok, thanks good to know, I changed it.
By the way I run the serialization against our archive (8.5 Mil files) and found some other parts that
throw errors. We have many dicom files with non conform values for the corresponding data types . For instance decimal with a value of 345. (single dot at the end with no following number) with produces an error, or instance numbers over int32. I fixed some of them should I make a pull request for those ?

gofal commented:
The strategy of fo-dicom is: Be as strict as possible when creating data, but be as tollerant as possible when reading data.
So the json serializer should only produce output that matches DICOM standard, but the json deserializer should be tollerant to be able to process invalid data. Of course fo-dicom should not guess some new data but should only correct values that are obvious. 

A pull request would be great for that!
